ONSdigital/ssdc-rm-response-operations
# 3036 rops define sample validation via create survey

# Motivation and Context
Cutting and pasting huge chunks of json validation rules is both messy and uncontrolled.
Instead the validation rules already exist in github, get them from there.

# What has changed
Added in survey Type

# How to test?
mvn clean install
Then run it and in the front end you should be able/forced to create a survey with a survey type.  Check the DB, do the rules look sensible


# Links
[

](https://trello.com/b/yarvSxGu/social-generic-response-management)

# Screenshots (if appropriate):
